man/resolved.conf.xml

```diff
@@ -335,6 +335,13 @@ DNSStubListenerExtra=udp:[2001:db8:0:f102::13]:9953</programlisting>
         url="https://www.iab.org/documents/correspondence-reports-documents/2013-2/iab-statement-dotless-domains-considered-harmful/">IAB
         Statement</ulink>, and may create a privacy and security risk.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>StaleRetentionSec=<replaceable>SECONDS</replaceable></term>
+        <listitem><para>Takes a duration value to determine for how long DNS resource records can be retained in the cache beyond their Time To Live (TTL) to return them as stale records.
+        This is useful when a DNS server failure occurs or not reachable, as systemd-resolved will continue to use these stale records to answer the DNS questions in case of a negative response from the upstream servers.
+        This improves the resilience against DNS infrastructure failures and outages.
+        However, this doesn't apply to NXDOMAIN responses, as those are still perfectly valid responses.</para></listitem>
```

Comment:
should mention the default value if not specified, and make clean what setting this to 0 means, and what setting this to infinity means

Reply:
also, please add an appropriate line to src/resolve/resolved.conf.in, commented out with an `#` just like the other lines, with the default value. This is useful for people who modify configuration as they can immediately see what the default is, and what the concept is called.

Reply:
Updated and #StaleRetentionSec=86400 to src/resolve/resolved.conf.in